playwright/tests/page-objects/control-bar.ts

```diff
@@ -18,7 +18,7 @@ export class ControlBar {
   }
 
   async toggleBrowser() {
-    const button = this.bar.getByRole('switch', { name: 'Simulate' });
+    const button = this.bar.getByLabel('Open Simulation').getByRole('button');
```

Comment:
You didn't apply all my changes, read carefully
getByRole('button', {name: 'Open Simulation'})